pcdsdevices/epics_motor.py

```diff
@@ -376,6 +376,13 @@ def flag_is_cleared(value=None, **kwargs):
             status_wait(st, timeout=timeout)
         return st
 
+    def camonitor(self):
+        try:
+            while True:
+                print("\r {0:4f}".format(self.user_readback.value), end=" ")
```

Comment:
This is why the cursor is flashing across the screen so fast. Python is racing through this `while` loop as fast as it can and repeatedly printing to the screen at max speed. We should either slow it down with a `time.sleep` call or implement this as a subscription callback e.g.
```
def cb(value=None, **kwargs):
    print("\r {0:4f}".format(value), end=" ")
cbid = self.user_readback.subscribe(cb)
```
And then later at the end, probably in a `finally` block:
`self.user_readback.unsubscribe(cbid)`